packages/coinstac-ui/app/main/index.js

```diff
@@ -238,6 +238,19 @@ loadConfig()
         });
     });
 
+    /**
+   * IPC Listener to stop pipeline
+   * @param {String} pipelineId The id of the pipeline currently running
+   * @param {String} runId The id of the pipeline run
+   * @return {Promise<String>} Status message
+   */
+
+    ipcMain.on('stop-pipeline', (event, { pipelineId, runId }) => {
+      return core.stopPipeline(pipelineId, runId)
+      .catch((err) => { console.log(`Pipeline could not be stopped because of the following error: ${err}`)});
```

Comment:
this should probably be logged using the winston instance and then sent to the UI toast handler